config/jobs/kubernetes/node-problem-detector/node-problem-detector-ci.yaml

```diff
@@ -94,13 +111,21 @@ periodics:
         '--node-test-args=--container-runtime-endpoint=unix:///run/containerd/containerd.sock --container-runtime-process-name=/home/containerd/usr/local/bin/containerd --container-runtime-pid-file= --kubelet-flags="--cgroup-driver=systemd --cgroups-per-qos=true --cgroup-root=/ --runtime-cgroups=/system.slice/containerd.service" --extra-log="{\"name\": \"containerd.log\", \"journalctl\": [\"-u\", \"containerd\"]}"'
         '--test_args=--nodes=8 --focus=NodeProblemDetector'
         --timeout=60m
+      resources:
+        limits:
+          cpu: 2
+          memory: 4Gi
+        requests:
+          cpu: 2
+          memory: 4Gi
   annotations:
     testgrid-dashboards: sig-node-node-problem-detector
     testgrid-alert-email: zhenw@google.com,lantaol@google.com
     testgrid-num-failures-to-alert: '12'
     testgrid-alert-stale-results-hours: '24'
     testgrid-num-columns-recent: '30'
 - name: ci-npd-e2e-kubernetes-gce-gci
+  cluster: eks-prow-build-cluster
```

Comment:
https://prow.k8s.io/job-history/gs/kubernetes-jenkins/logs/ci-npd-e2e-kubernetes-gce-gci-custom-flags?buildId=1675443561614544896

```
2023/07/01 14:01:41 main.go:328: Something went wrong: failed to prepare test environment: --provider=gce boskos failed to acquire project: resources not found
Traceback (most recent call last):
  File "/workspace/scenarios/kubernetes_e2e.py", line 723, in <module>
    main(parse_args())
  File "/workspace/scenarios/kubernetes_e2e.py", line 569, in main
    mode.start(runner_args)
  File "/workspace/scenarios/kubernetes_e2e.py", line 228, in start
    check_env(env, self.command, *args)
  File "/workspace/scenarios/kubernetes_e2e.py", line 111, in check_env
    subprocess.check_call(cmd, env=env)
  File "/usr/lib/python3.9/subprocess.py", line 373, in check_call
    raise CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command '('kubetest', '--dump=/logs/artifacts', '--gcp-service-account=/etc/service-account/service-account.json', '--up', '--down', '--test', '--provider=gce', '--cluster=e2e-a52e116b34-75293', '--gcp-network=e2e-a52e116b34-75293', '--extract=ci/latest', '--gcp-node-image=gci', '--gcp-zone=us-west1-b', '--ginkgo-parallel=30', '--test_args=--ginkgo.focus=NodeProblemDetector', '--timeout=60m')' returned non-zero exit status 1.
+ EXIT_VALUE=1
+ set +o xtrace
```

Some test failed after this PR merged.

Reply:
@pacoxu From the original issue:

> NOTE: if you see any entries under label that says gce skip this job and go to the next time as this is not ready to be moved yet.

The test name contains `gce` but it's not present in labels, could this be the problem?

Reply:
I opened https://github.com/kubernetes/test-infra/issues/29998.  I am not quite sure about that.